### DISCUSSION TITLE
Alpha 187

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#7529] [Release Notes]
# Changes
- d038ac6: Added Swedish translation
- 875e693: Improved pane rearranging UI

# Fixes
- d26ce6c: Fixed broadcast / focus all command being unavailable for SSH tabs - #7481
- 0f0f61f: SSH: correctly handle MaxSessions when multiplexing - #6263, #7460 
- 9c89bab: Fixed Zmodem errors - #6677, #5845, #5243, #5132, #5021, #7511, #7053, #6917, #6639, #6259, #6182, #6122, #5845, #5737, #5701, #5609, #5311, #5243, #5231, #5132

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/tabby/releases/tag/v1.0.187'>Alpha 187</a>.</em>

**Comment by hhr114** [comment]
better than version 19x
